src/Compilers/CSharp/Portable/Syntax/InternalSyntax/SyntaxToken.MissingTokenWithTrivia.cs

```diff
@@ -37,6 +37,26 @@ public override string Text
                 get { return string.Empty; }
             }
 
+            public override string ValueText
```

Comment:
>ValueText [](start = 35, length = 9)

This is very funky. I don't think we should do this.
I'll try to offer a suggestion.

Reply:
I agree with Julien.  This breaks invariants the syntax model.